keps/sig-auth/5018-dra-adminaccess/README.md

```diff
@@ -466,7 +466,7 @@ ResourceClaimTemplate and ResourceClaim for admin access
 
 - Gather feedback
 - Additional tests are in Testgrid and linked in KEP
-- Implementations in the kubernetes-sigs/dra-example-driver
+- Implementations in the kubernetes-sigs/dra-example-driver: https://github.com/kubernetes-sigs/dra-example-driver/issues/97 and the NVIDIA dra driver: https://github.com/NVIDIA/k8s-dra-driver-gpu/issues/337
```

Comment:
do those issues mean we will show those repos labeling namespaces as admin access and using devices as admin access before promoting the gate to beta?

Reply:
Yes, `Implementations in the kubernetes-sigs/dra-example-driver` was part of the original beta criteria. I think we should be able to add an example there. I'm less certain about the exact timeline of the Nvidia one. I could remove that one for now and add it back AFTER it's done. wdyt?

Reply:
let's not take dependencies on consumers we're not sure will be ready as a beta graduation criteria ... one example use seems sufficient